internal/database/fakedb/users.go

```diff
@@ -44,3 +45,19 @@ func (users *Users) GetByCurrentAuthUser(ctx context.Context) (*types.User, erro
 	}
 	return a.User(ctx, users)
 }
+
+func (users *Users) List(ctx context.Context, opts *database.UsersListOptions) (_ []*types.User, err error) {
```

Comment:
nit: I don't think the `err` value is referred to in this function.

```suggestion
func (users *Users) List(ctx context.Context, opts *database.UsersListOptions) ([]*types.User, error) {
```